packages/components/pie-modal/src/index.ts

```diff
@@ -105,15 +109,15 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
     connectedCallback () : void {
         super.connectedCallback();
         this.addEventListener('click', (event) => this._handleDialogLightDismiss(event));
-        document.addEventListener(ON_MODAL_OPEN_EVENT, this._handleModalOpened.bind(this));
-        document.addEventListener(ON_MODAL_CLOSE_EVENT, this._handleModalClosed.bind(this));
-        document.addEventListener(ON_MODAL_BACK_EVENT, this._handleModalClosed.bind(this));
+        document.addEventListener(ON_MODAL_OPEN_EVENT, (event) => this._handleModalOpened(<CustomEvent>event));
```

Comment:
Interesting! Would this be equivalent to something like this? (I'm not suggesting you change it, just asking 😄) 
```suggestion
        document.addEventListener(ON_MODAL_OPEN_EVENT, (event : CustomEvent) => this._handleModalOpened(event));
```

Reply:
Yeah so it's basically the same but two different ways (Type assertion which is what i have & Type annotation). I did try your way first but i had a IDE warning so i went the other way which seems to stop that warning.